res/database/Default/user/scr0llbaer_constr_d2v_tv_series.xml

```diff
@@ -0,0 +1,6397 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tvtdb>
+	<version value="3" comment="Direct-to-TV, TV movies and series entries under construction. Incomplete, not validated, and not tested. For build-stage review, feedback and coordination with other contributors only. Contents will be moved to appropriate places, and this file will be deleted in the last step of the PR." />
+
+	<allprogrammes>
+
+		<!-- fictional -->
+		
+		<programme id="dcf404f3-7f17-4f4f-83cd-83626951a4f0" product="4" licence_type="1" fictional="1" creator="9551">
+			<title>
+				<de>Dynamit - Das Magazin Live: Hochzeit von [3|First] und [2|Full]</de>
+				<en>Dynamite - The Magazine Live: Wedding of [3|First] und [2|Full]</en>
+			</title>
+			<description>
+				<de>Der 53-jährige Fürst [2|Full] und die 20-jährige Punk-Pinzessin [3|First] heiraten in Regensburg, ja kann das gut gehen? Wie schön ist's im Schloss? Ist es wirklich Liebe? Braucht der Fürst Hilfsmittel zum Schnackseln? Fragen über Fragen.</de>
+				<en>The 53-year-old Prince [2|Full] and the 20-year-old Punk Princess [3|First] get married in Regensburg, can that go well? How beautiful is it in the castle? Is it really love? Does the prince need aids for bonking? Questions upon questions.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">cebd1a68-cf97-4b93-b3f9-ac34bda4bdcc</member>
+				<member index="1" function="136">88ba23dc-d4f1-4c77-9ddf-8dbf4a4ce3b2</member>
+				<member index="2" function="64">9d987fbb-99c3-47b8-82c3-227461b6762c</member>
+				<member index="3" function="64">64828f7e-a96b-4bb3-85c6-477ebe408d05</member>
+			</staff>
+			<data country="D" distribution="2" maingenre="301" subgenre="6,15" flags="17" blocks="6" price_mod="0.5" />
+			<releaseTime year="1980" day="5" hour="10" />
```

Comment:
I followed the example of other live shows like `ronny-programme-livereportage-raketenstart1`

I think we don't have enough of this "boulevard" format so I thought it shouldn't disappear when not starting in 1980, and so that it can be a cheap morning programme re-run.